signing/sawtooth_signing/core.py

```diff
@@ -128,6 +128,22 @@ def new_random_private_key(self):
             (:obj:`PrivateKey`): a random private key
         """
 
+    @abstractmethod
+    def private_key_from_bytes(self, bytes):
+        """Generates a new random PrivateKey using this context.
+
+        Returns:
+            (:obj:`PrivateKey`): a privatekey
+        """
+
+    @abstractmethod
+    def private_key_from_hex(self, hex):
```

Comment:
While I think the at least the `from_bytes` method is absolutely necessary, there is some debate about whether or not hex is too opinionated to use. My inclination is to keep it, but I am flagging this because it has been an area of discussion in the past.